rust/operator-binary/src/spark_k8s_controller.rs

```diff
@@ -155,6 +155,12 @@ pub enum Error {
     CreateVolumes {
         source: stackable_spark_k8s_crd::Error,
     },
+
+    #[snafu(display("Failed to update status for application [{name}]"))]
```

Comment:
Since `name` is a String, I would then use the Debug impl which adds quotes.

```suggestion
    #[snafu(display("Failed to update status for application {name:?}"))]
```